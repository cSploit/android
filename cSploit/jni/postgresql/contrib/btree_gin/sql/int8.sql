set enable_seqscan=off;

CREATE TABLE test_int8 (
	i int8
);

INSERT INTO test_int8 VALUES (-2),(-1),(0),(1),(2),(3);

CREATE INDEX idx_int8 ON test_int8 USING gin (i);

SELECT * FROM test_int8 WHERE i<1::int8 ORDER BY i;
SELECT * FROM test_int8 WHERE i<=1::int8 ORDER BY i;
SELECT * FROM test_int8 WHERE i=1::int8 ORDER BY i;
SELECT * FROM test_int8 WHERE i>=1::int8 ORDER BY i;
SELECT * FROM test_int8 WHERE i>1::int8 ORDER BY i;
