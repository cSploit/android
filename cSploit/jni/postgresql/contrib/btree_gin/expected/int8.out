set enable_seqscan=off;
CREATE TABLE test_int8 (
	i int8
);
INSERT INTO test_int8 VALUES (-2),(-1),(0),(1),(2),(3);
CREATE INDEX idx_int8 ON test_int8 USING gin (i);
SELECT * FROM test_int8 WHERE i<1::int8 ORDER BY i;
 i  
----
 -2
 -1
  0
(3 rows)

SELECT * FROM test_int8 WHERE i<=1::int8 ORDER BY i;
 i  
----
 -2
 -1
  0
  1
(4 rows)

SELECT * FROM test_int8 WHERE i=1::int8 ORDER BY i;
 i 
---
 1
(1 row)

SELECT * FROM test_int8 WHERE i>=1::int8 ORDER BY i;
 i 
---
 1
 2
 3
(3 rows)

SELECT * FROM test_int8 WHERE i>1::int8 ORDER BY i;
 i 
---
 2
 3
(2 rows)

